.evergreen/config.yml

```diff
@@ -588,17 +589,18 @@ buildvariants:
     - ubuntu1804-small
   tasks:
      - name: "test"
-- matrix_name: "topologies-old"
-  matrix_spec:
-    driver: current
-    ruby: "ruby-2.6"
-    mongodb-version: ['2.6', '3.0', '3.2', '3.4']
-    topology: ["replica-set", "sharded-cluster"]
-  display_name: "${ruby}, ${driver}, ${mongodb-version}, ${topology}"
-  run_on:
-    - ubuntu1404-small
-  tasks:
-     - name: "test"
+# https://jira.mongodb.org/browse/MONGOID-5133
+# - matrix_name: "topologies-old"
+#   matrix_spec:
+#     driver: current
+#     ruby: "ruby-2.6"
+#     mongodb-version: ['2.6', '3.0', '3.2', '3.4']
+#     topology: ["replica-set", "sharded-cluster"]
+#   display_name: "${ruby}, ${driver}, ${mongodb-version}, ${topology}"
+#   run_on:
+#     - ubuntu1404-small
+#   tasks:
+#      - name: "test"
```

Comment:
Better to just delete rather than commenting it out?

Reply:
Not in this case, I believe.